mlflow/utils/crypto.py

```diff
@@ -145,24 +162,15 @@ def __init__(self, passphrase: str | None = None, kek_version: int | None = None
         if kek_version is None:
             kek_version = int(os.getenv(CRYPTO_KEK_VERSION_ENV_VAR, "1"))
 
+        # Use default passphrase if none provided
+        self._using_default_passphrase = not passphrase
         if not passphrase:
-            raise MlflowException(
-                "MLFLOW_CRYPTO_KEK_PASSPHRASE environment variable is required for "
-                "secrets management.\n\n"
-                "This passphrase is used to encrypt secrets in the database and must be set "
-                "on the MLflow server.\n\n"
-                "SECURITY: Use a high-entropy passphrase (32+ characters) from your secrets "
-                "manager to ensure strong protection.\n\n"
-                "For local development:\n"
-                "  export MLFLOW_CRYPTO_KEK_PASSPHRASE='my-dev-secret-at-least-32-chars'\n\n"
-                "For production (use your secrets manager):\n"
-                "  export MLFLOW_CRYPTO_KEK_PASSPHRASE='$(vault kv get "
-                "-field=passphrase mlflow/kek)'\n\n"
-                "For Kubernetes:\n"
-                "  Store passphrase in a Secret and inject via secretKeyRef\n\n"
-                "IMPORTANT: Users do NOT need this passphrase - only server administrators.\n"
-                "Users connect to the MLflow server over HTTPS and create secrets normally.",
-                error_code=INVALID_PARAMETER_VALUE,
+            passphrase = DEFAULT_KEK_PASSPHRASE
+            _logger.warning(
+                "MLFLOW_CRYPTO_KEK_PASSPHRASE not set. Using default passphrase for "
+                "secrets encryption. This is acceptable for development/testing but "
```

Comment:
Shall we explicitly tell what kind of case is risky? Here I think development/testing refers to localhost tracking server and production refer to remove tracking server (or tracking server with public hostname), correct? Even for users who share a remote tracking server among multiple team members for agent development, we want them to set `MLFLOW_CRYPTO_KEK_PASSPHRASE`